src/ctia/http/routes/observable.clj

```diff
@@ -48,19 +46,28 @@
 
   (GET "/:observable_type/:observable_value/indicators" []
     :tags ["Indicator"]
-    :query [params IndicatorsByObservableQueryParams]
+    :query [params IndicatorRefsByObservableQueryParams]
     :path-params [observable_type :- ObservableType
                   observable_value :- s/Str]
-    :return (s/maybe [StoredIndicator])
-    :summary "Returns all the Indicators associated with the specified observable."
+    :return (s/maybe [c/Reference])
+    :summary "Returns all the Indicator References associated with the specified observable."
     :header-params [api_key :- (s/maybe s/Str)]
     :capabilities #{:list-judgements :list-indicators}
     (paginated-ok
-     (as-> {:type observable_type
-            :value observable_value} $
-       (list-judgements-by-observable @judgement-store $ nil)
-       (:data $ [])
-       (list-indicators-by-judgements @indicator-store $ params))))
+     (let [res (as-> {:type observable_type
+                      :value observable_value} $
+                 (list-judgements-by-observable @judgement-store $ nil)
+                 (:data $ [])
+                 (mapcat :indicators $)
+                 (map :indicator_id $)
+                 (set $) ;; remove duplicate URIs
+                 (vec $)
+                 (sort $))]
```

Comment:
@yogsototh 
I recommend using `->>`. Also, note the two things are unneeded. 
1) don't need the `[]` arg to `(:data $ [])`  (which enables everything to be a `->>`), because it'll default to a nil result which you can `mapcat` over no problem.
2) don't need the `vec` call, which immediately gets turned into a non-vector type when you call `sort`

SEE, for example:
```clj
(class (sort [1 2 3]))
;;=> clojure.lang.ArraySeq
```

MY VERSION:
```clj
(->> (list-judgements-by-observable @judgement-store {:type observable_type
                                                      :value observable_value} nil)
     :data
     (mapcat :indicators)
     (map :indicator_id)
     distinct ;; remove duplicate URIs
     sort)
```